azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java

```diff
@@ -250,6 +250,13 @@ public Status getStartStatus() {
     return Status.PREPARING;
   }
 
+  /**
+   * @return Return the start status based on the {@link ExecutableFlow}
+   */
+  public Status getStartStatus(ExecutableFlow execFlow) {
```

Comment:
execFlow is never used here, is it supposed to call the new API?

Reply:
It is not used in the AbstractExecutorManagerAdapter, but for ContainerizedDispatchManager which overrides it implements it. Since the call to this made will be made on the class AbstractExecutorManagerAdapter, we need it here